pkgs/development/libraries/libgssglue/default.nix

```diff
@@ -1,14 +1,26 @@
-{ lib, stdenv, fetchurl, libkrb5 }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  libkrb5,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libgssglue";
-  version = "0.4";
+  version = "0.9";
 
-  src = fetchurl {
-    url = "http://www.citi.umich.edu/projects/nfsv4/linux/libgssglue/${pname}-${version}.tar.gz";
-    sha256 = "0fh475kxzlabwz30wz3bf7i8kfqiqzhfahayx3jj79rba1sily9z";
+  src = fetchFromGitLab {
+    owner = "gsasl";
+    repo = "libgssglue";
+    rev = "tags/libgssglue-${finalAttrs.version}";
+    hash = "sha256-p9dujLklv2ZC1YA1gKGCRJf9EvF3stv5v4Z/5m1nSeM=";
   };
 
+  preConfigure = "touch ChangeLog";
```

Comment:
looks like preConfigure happends after the check for if ChangeLog exists, sorry for the wrong info before,

Could you please move `touch ChangeLog` to the `postPatch` section as that runs before the check for the changelog is run.